txpipe/extensions/twopoint_cluster.py

```diff
@@ -0,0 +1,181 @@
+import sys
+import os
+import numpy as np
+import treecorr
+import h5py
+import sacc
+from txpipe.base_stage import PipelineStage
```

Comment:
The right syntax for importing from a different sub-package is to use a relative import:

```
from ..base_stage import PipelineStage
from ..data_types import HDFFile, SACCFile
```

I have updated this.